tortoise/queryset.py

```diff
@@ -450,6 +450,38 @@ def order_by(self, *orderings: str) -> "QuerySet[MODEL]":
         queryset._orderings = new_ordering
         return queryset
 
+    def latest(self, field_name: str = "id") -> "QuerySetSingle[Optional[MODEL]]":
+        """
+        Returns the most recent object by ordering descending on the specified field.
+
+        :params field_name: The field name to order by.
+        """
+        queryset = self._clone()
+        if not (
+            field_name.split("__")[0] in self.model._meta.fields or field_name in self._annotations
+        ):
+            raise FieldError(f"Unknown field {field_name} for model {self.model.__name__}")
+        queryset._orderings = [(field_name, Order.desc)]
+        queryset._limit = 1
+        queryset._single = True
+        return queryset  # type: ignore
+
+    def earliest(self, field_name: str = "id") -> "QuerySetSingle[Optional[MODEL]]":
+        """
+        Returns the earliest object by ordering ascending on the specified field.
+
+        :params field_name: The field name to order by.
+        """
+        queryset = self._clone()
+        if not (
```

Comment:
Can you please add few test about passing related fields there? 
Like `related_model__pub_date`

I understand that it is not a prime usecase for that, but if we are supporting that behaviour - would be nice to have it covered

Reply:
I've added more tests to increase coverage, including testing the use of related fields `tests/test_latest_earliest.py`